pages/desktop/user.py

```diff
@@ -112,6 +113,10 @@ def details_header_text(self):
         return self.selenium.find_element(*self._details_locator).text
 
     @property
+    def checkbox_status(self):
```

Comment:
This name seems to vague for what the property is. This returns `True` if the checkbox is checked and `False` if it is not, but I wouldn't intuit that from the name `checkbox_status`. Perhaps something like `is_hide_email_checked` would be more descriptive.

Having said that, I don't think we actually need this method at all as the test doesn't really need it if the page object handles `hide_email_address` appropriately.